PocketKit/Sources/PocketKit/Item/Item+Extensions.swift

```diff
@@ -40,6 +40,10 @@ public extension Item {
                 // then we should open in web view
                 return true
             } else {
+                // if the item is an article and has no article components (marticle), then open in web view
+                if isArticle && !hasArticleComponents {
```

Comment:
@cyndichin theres a case (ike archive results) where we purposedly did not download marticle ocmponents, but we download them on demand. Will this ensure that if that download fails then we fall back to webview

Reply:
ah okay, no wonder tests were breaking for archive and syndicated, I updated to use the logic in our readableviewmodels instead. thanks!